app-core-test/src/test/java/net/spals/appbuilder/app/core/generic/SampleGenericWorkerAppFTest.java

```diff
@@ -169,4 +170,10 @@ public void testModelInjection() {
         assertThat(modelSerializerMap, aMapWithSize(1));
         assertThat(modelSerializerMap, hasKey("pojo"));
     }
+
+    @Test
+    public void testMonitorInjection() {
+        final Injector serviceInjector = sampleApp.getServiceInjector();
+        assertThat(serviceInjector.getInstance(Tracer.class), instanceOf(NoopTracer.class));
```

Comment:
Tests that the `NoopTracer` is used as the default within a `GenericWorkerApp`

Reply:
FWIW I believe some people consider with the @Test flag its redundant to use the "test" prefix which was required for junit 3 to figure out tests.

Reply:
Oh, is that true about junit 3? I just adopted that standard from the tests I saw early at SFDC.

Reply:
Yeah it was required before java annotations as it does a reflective search for methods prepended with "test"